public/partials/layer/view.html

```diff
@@ -89,7 +89,7 @@ <h3 app-editable-text="_Layer.name">{{ _Layer.name }}</h3>
                                            <i class="fa fa-cube wst-main-color" ng-show="rootItem.elementType != 'date' && rootItem.elementType != 'number' && rootItem.elementType != 'count' && (!rootItem.defaultAggregation)" data-toggle="tooltip" data-placement="top" title="{{rootItem.elementLabel}} (Dimmension)"></i>
                                                         <i class="fa fa-reorder wst-main-color" ng-show="rootItem.elementType == 'number' || rootItem.elementType == 'count' || (rootItem.elementType == 'string' && rootItem.defaultAggregation)" data-toggle="tooltip" data-placement="top" title="{{rootItem.elementLabel}} (Metric)"></i>
                                                         <i class="fa fa-calendar-o wst-main-color" ng-show="rootItem.elementType == 'date'" data-toggle="tooltip" data-placement="top"></i>
-                                           <span ng-if="!rootItem.editing == true">{{rootItem.elementLabel}}</span>
+                                           <span ng-if="!rootItem.editing == true" data-toggle="tooltip" data-placement="top" title="{{rootItem.description}}">{{rootItem.elementLabel}}</span>
```

Comment:
Does the `data-toggle` and `data-placement` attributes have any effect ? I tried removing them and I saw no differences